CurrencyDotNet.Common/Extensions/StringExtensions.cs

```diff
@@ -27,5 +27,19 @@ public static string HmacSha256(this string message, string secret)
 
             return BitConverter.ToString(bytes).Replace("-", "").ToLower();
         }
+
+        /// <summary>
+        /// Returns the key value of the request Uri parameter
+        /// </summary>
+        /// <param name="value">Input value</param>
+        /// <returns>Key value of the request Uri parameter</returns>
+        public static string GetUriRequestParameterKey(this string value)
+        {
+            if (value.Equals("ApiKey")) return "X-MBX-APIKEY";
+
+            var firstPartLower = char.ToLower(value[0]);
+            var secondPart = value[1..];
```

Comment:
Interesting syntax, I've never seen before something like this. Why did you decide to use this instead of substring (It is just question)? 
Is it work correct in case when value length equals 1?

Reply:
The calculation here is that properties in a class cannot have a single character name. I think that if a class property has a bad name, it's a bad class.